workshops/views.py

```diff
@@ -554,13 +554,33 @@ def person_bulk_add(request):
                     # the data
                     request.session['bulk-add-people'] = persons_tasks
                     return redirect('person_bulk_add_confirmation')
+        elif form.is_valid() and form.cleaned_data['event']:
+            event = form.cleaned_data['event']
+            try:
+                r = requests.get(event.url + '/api/speaker/')
```

Comment:
This assumes that the url is saved with no trailing '/'.
What is the accepted practice here?
Do I override the `save()` method of the `Person` model to ensure such a constraint?